src/test/java/com/ericsson/ei/handlers/test/ObjectHandlerTest.java

```diff
@@ -92,9 +92,8 @@ public void init() throws Exception {
 
     @Test
     public void test() {
```

Comment:
Write meaningful test name to this test, e.g. something about what it tests.

Reply:
done.... I hope the new name is meaningful